apps/system/init/init.c

```diff
@@ -133,6 +139,13 @@ int preapp_start(int argc, char *argv[])
 	}
 #endif
 
+#ifdef CONFIG_MEDIA
+	ret = media_init();
```

Comment:
media_init funtion is built with CONFIG_AUDIO. But here, you add it with CONFIG_MEDIA. Please check it

Reply:
Thanks for review. I fixed it in #2593